backend/app/controllers/spree/admin/refunds_controller.rb

```diff
@@ -8,6 +8,22 @@ class RefundsController < ResourceController
 
       rescue_from Spree::Core::GatewayError, with: :spree_core_gateway_error
 
+      def create
+        @refund.attributes = refund_params
+        if @refund.save
+          @refund.perform!
```

Comment:
@jordan-brough 
Changed to `#perform_and_save!` as per suggestions from @jhawthorn and @cbrunsdon 

Hopefully this appeases all parties!